hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -82,10 +63,25 @@ After five to ten seconds, the Raspberry Pi should begin booting and show the ra
 
 See the [bootmodes documentation](README.md) for the boot sequence and alternative boot modes (network, USB device, GPIO or SD boot).
 
-## Known issues (not Pi 4B, CM4 and Pi 400)
+## Known issues (not Raspberry Pi 4)
 
 - The default timeout for checking bootable USB devices is 2 seconds. Some flash drives and hard disks power up too slowly. It is possible to extend this timeout to five seconds (add a new file `timeout` to the SD card), but note that some devices take even longer to respond.
 - Some flash drives have a very specific protocol requirement that is not handled by the bootcode and may thus be incompatible.
 
-## Special bootcode.bin-only boot mode (not Pi 4B, CM4 and Pi 400)
+## Special bootcode.bin-only boot mode (not Raspberry Pi 4)
 If you are unable to use a particular USB device to boot your Raspberry Pi, an alternative for the Pi 2B v1.2, 3A+, 3B and 3B+ is to use the special bootcode.bin-only boot mode as described [here](README.md). The Pi will still boot from the SD card, but `bootcode.bin` is the only file read from it.
+
+## Hardware compatibility for USB mass storage devices
+Before attempting to boot from a USB mass storage device it is advisable to verify that the device works correctly under Linux. Boot using an SD card and plug in the USB mass storage device. This should appears as a removable drive.
+
+*Spinning hard-disk drives nearly always require a powered USB hub. Even if it appears to work you are likely to encounter intermittent failures without a powered USB HUB*
```

Comment:
I would change `hard-disk drives` to to `hard disks` (less wordy), or simply remove the hyphen.

Also, the docs use bold for emphasis. Perhaps just put a bold `Note:` at the start of the sentence and remove the italics.

And the last sentence needs a full stop on the end.

And this paragraph appears to have been inserted in the wrong place, since the paragraphs above and below it appear to be related. I would therefore move this paragraph to the end of this section (i.e. below `See this forum thread...`).

Finally, I would merge the first paragraph with the one which starts `This is especially important...` since thee use of "this" to refer to the previous sentence means that gramatically the two sentences should be in the same paragraph.

Reply:
removed emphases and reordered sentances